### DISCUSSION TITLE
Création d'un profil « consultation seule » pour les lycées

### DISCUSSION THREAD

**Discussion by MagaliMoserNoriaIRL** [unistra/esup-immersup-demandes#12] [6- Administration]
Création d'un profil « consultation seule » comme pour les universités

**Comment by efinkbeiner5** [comment]
Est-ce que cela vaut que pour les lycées partageant leur offre ? 
Pour la gestion des créneaux ? 
Compléxité : Moyen

**Reply by efinkbeiner5** [reply]
Pour les lycées partageant leur offre

**Comment by efinkbeiner5** [comment]
Comme le profil CONS_STR (Consultation Structure), créer un profil pour les lycées partageant leur offre permettant aux accueils des lycées d'avoir la liste des créneaux et les inscrits pour renseigner les lycéens des autres lycées qui viendraient en immersion. 
Complexité : modérée 
Chiffrage : 4jours de développement 
Il faut lister toutes les pages auxquelles il devra accéder

**Comment by efinkbeiner5** [comment]
Non réalisé en 2025

**Comment by MagaliMoserNoriaIRL** [comment]
Bonjour. J'aimerais remettre cette fonctionnalité dans la liste des demandes :)
